setup.py

```diff
@@ -51,10 +51,10 @@
     },
     install_requires=[
         'matplotlib>=3.0,<3.1',
-        'numpy>=1.19.2,<1.20',
+        'numpy==1.19.2,<1.20',
```

Comment:
This looks strange to me. If we're insisting on `numpy==1.19.2`, why do we need to also say `<1.20`?

Also, I don't know why we would be so specific about numpy as to not accept 1.19.x bugfixes.

Having said that, I am sure the versions need tweaking.

Reply:
Hi @douglasbagnall, 

Thanks. This is definitely wrong and it is a typo (I was trying different combinations at some point). I will fix this.

Laurent